cmd/main.go

```diff
@@ -4,16 +4,21 @@
 package main
 
 import (
-	"fmt"
+	"flag"
 	"net/http"
 
 	"github.com/ga4gh/htsget-refserver/internal/htsconfig"
 	"github.com/ga4gh/htsget-refserver/internal/htsserver"
+	log "github.com/sirupsen/logrus"
```

Comment:
```suggestion
```

Always start form top of main in order to prevent PR conflicts.

Reply:
This is from the top of main! the order of PRs can matter - but in this case it's probably easier to rebase some other PR on this than the other way around.

Reply:
https://github.com/NBISweden/htsget-refserver/blob/197f3f50cdb735e523fe1f88b9af0f18faec3a0b/cmd/main.go#L6-L12

Reply:
What do you mean?